### PR TITLE
Add extension for CI flake8

### PR DESCRIPTION
**Pull request by lyriccoder**
Extension for CI flake8

Fix https://github.com/cqfn/veniq/issues/2

### CODE REVIEW

**Comment by lyriccoder** [comment]
@acheshkov could you please set the number of approving reviews to 2, cause we have CI as approver:
https://docs.github.com/en/github/administering-a-repository/enabling-required-reviews-for-pull-requests

**Comment by acheshkov** [comment]
> @acheshkov could you please set the number of approving reviews to 2, cause we have CI as approver:
> https://docs.github.com/en/github/administering-a-repository/enabling-required-reviews-for-pull-requests

okay

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    steps:
+      - uses: actions/checkout@v1
+      - uses: grantmcconnaughey/lintly-flake8-github-action@v1.0
+        with:
+          # The GitHub API token to create reviews with
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # Fail if "new" violations detected or "any", default "new"
+          failIf: new
+          # Additional arguments to pass to flake8, default "." (current directory)
+          args: "--ignore=E121"
```
Comment (acheshkov) [inline .github/workflows/blank.yml:23]: why do you ignore E121?

File: .github/workflows/blank.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: grantmcconnaughey/lintly-flake8-github-action@v1.0
+        with:
+          # The GitHub API token to create reviews with
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (acheshkov) [inline .github/workflows/blank.yml:19]: How GITHUB secret token works?

File: .github/workflows/blank.yml (hunk 1 above)
Comment (lyriccoder) [reply .github/workflows/blank.yml]: fixed

File: .github/workflows/blank.yml (hunk 2 above)
Comment (lyriccoder) [reply .github/workflows/blank.yml:19]: https://docs.github.com/en/actions/reference/authentication-in-a-workflow

**Review by acheshkov — APPROVED** [review]

**Pull request merged**